app/forms/idv/api_image_upload_form.rb

```diff
@@ -320,6 +320,12 @@ def images_metadata
       @images_metadata ||= IdvImages.new(params)
     end
 
+    def doc_escrow_images
+      return {} unless service_provider.attempts_api_enabled?
```

Comment:
another option is to put this code into the `doc_escrow_enabled?` [method](https://github.com/18F/identity-idp/blob/main/app/forms/idv/api_image_upload_form.rb#L154-L156).